src/VisualStudio.Roslyn.SDK/Roslyn.SDK/ItemTemplates/CSharp/Analyzer/CSharpAnalyzer.vstemplate

```diff
@@ -11,4 +11,14 @@
   <TemplateContent>
     <ProjectItem SubType="Code" TargetFileName="$fileinputname$.cs" ReplaceParameters="true">Analyzer.cs</ProjectItem>
   </TemplateContent>
+  <WizardExtension>
+    <Assembly>NuGet.VisualStudio.Interop, Version=1.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</Assembly>
+    <FullClassName>NuGet.VisualStudio.TemplateWizard</FullClassName>
+  </WizardExtension>
+  <WizardData>
+    <packages>
+      <package id="Microsoft.CodeAnalysis.Workspaces.Common" version="3.2.0" />
```

Comment:
❗ The workspaces layer cannot be added to an analyzer project. Code fixes (which use the workspaces layer) go in a different assembly from analyzers.

```suggestion
```